aksel.nav.no/website/app/_ui/code-block/CodeBlock.tsx

```diff
@@ -136,16 +131,41 @@ function CodeBlockEditor(props: {
             <code>
               {tokens.map((line, i) => {
                 const lineProps = getLineProps({ line });
+                const lineText = line.map((t) => t.content).join("");
+                const lineMark = getLineMark(lang, lineText);
+
                 return (
                   <div
                     key={i}
                     {...lineProps}
                     style={{ ...lineProps.style, "--line": `"${i + 1}"` }}
                     className={styles.codeBlockLine}
+                    data-line-mark={lineMark}
                   >
-                    {line.map((token, key) => (
-                      <span key={key} {...getTokenProps({ token })} />
-                    ))}
+                    {line.map((token, key) => {
+                      let content = token.content;
+                      if (key === 0 && lineMark) {
+                        content = content.replace(/^[+->]/, "");
```

Comment:
This could be simplified (unless we want to make it clear which characters we are removing? 🤔)

```suggestion
                        content = content.slice(1);
```